modules/imports/src/test/scala/org/dhallj/imports/ImportResolutionSuite.scala

```diff
@@ -148,9 +148,9 @@ class ImportResolutionSuite extends FunSuite {
   test("Valid hash") {
 
     val expr = parse(
-      "let x = classpath:/hashed/package.dhall sha256:d60d8415e36e86dae7f42933d3b0c4fe3ca238f057fba206c7e9fbf5d784fe15 in x"
+      "classpath:/hashed/package.dhall sha256:d60d8415e36e86dae7f42933d3b0c4fe3ca238f057fba206c7e9fbf5d784fe15"
     )
-    val expected = parse("let x = 1 in x").normalize
+    val expected = parse("let x = 1 in x").alphaNormalize.normalize
```

Comment:
What's up here? It looks a little odd to alpha-normalize before beta-normalizing.

Reply:
Is it? In my head alpha-normalization belongs at the same stage as computing De Bruijn indices (although obviously not necessary in the same way as Dhall variables are unambiguous)